meeteval/wer/wer/mimo.py

```diff
@@ -99,28 +100,30 @@ def mimo_word_error_rate(reference, hypothesis) -> MimoErrorRate:
     >>> mimo_word_error_rate(STM.parse('X 1 A 0.0 1.0 a b\\nX 1 A 1.0 2.0 c d\\nX 1 B 0.0 2.0 e f\\n'), STM.parse('X 1 1 0.0 2.0 c d\\nX 1 0 0.0 2.0 a b e f\\n'))
     MimoErrorRate(error_rate=0.0, errors=0, length=6, insertions=0, deletions=0, substitutions=0, assignment=[('A', '0'), ('B', '0'), ('A', '1')])
     """
-    reference = asseglst(reference)
-    hypothesis = asseglst(hypothesis)
-
-    # Sort by start time if the start time is available
-    # TODO: implement something like reference_sort from time_constrained.py?
-    if 'start_time' in reference.T.keys():
-        reference = reference.sorted('start_time')
-    if 'start_time' in hypothesis.T.keys():
-        hypothesis = hypothesis.sorted('start_time')
+    reference, hypothesis, ref_self_overlap, hyp_self_overlap = preprocess(
```

Comment:
preprocess has no sort, or?

Reply:
No, the sorting is done in `api.py`. Removing this here actually fixes an issue where the sort option has no effect for mimo WER.

This at the same time changes the behavior of the (lower-level) python interface. But for this, it would make sense to add sorting to all non-time-constrained WERs, also to ORC-WER and cpWER.